docs/commerce/2.x/en/account-management/adding-organizations-to-accounts.md

```diff
@@ -0,0 +1,43 @@
+# Adding Organizations to Accounts
+
+There are two ways to add organizations to an account: 1) using the _Account Management_ widget and 2) through the _Control Panel_. Access to the _Control Panel_ is typically restricted to those with administrative permissions for the store or installation.
```

Comment:
Must be an existing organization. Unlike users which you can create on the spot, orgs have to be preexisting.